package.json

```diff
@@ -1,67 +1,79 @@
 {
-  "name": "mini-html-webpack-plugin",
-  "version": "0.0.0-development",
-  "description": "A miniature version of html-webpack-plugin with only necessary features",
-  "main": "index.js",
-  "files": [
-    "index.js"
-  ],
-  "scripts": {
-    "pretest": "npm run lint",
-    "test": "true && npm run test:jest",
-    "posttest": "npm run format",
-    "lint": "eslint . --cache --fix",
-    "format": "prettier --write \"**/*.{js,md}\"",
-    "test:jest": "jest",
-    "test:watch": "jest --watch",
-    "test:coverage": "jest --coverage"
-  },
-  "keywords": [
-    "webpack",
-    "html",
-    "template"
-  ],
-  "author": "Juho Vepsalainen",
-  "contributors": [
-    {
-      "name": "Artem Sapegin",
-      "url": "http://sapegin.me"
-    }
-  ],
-  "license": "MIT",
-  "dependencies": {
-    "webpack-sources": "^1.3.0"
-  },
-  "homepage": "https://github.com/styleguidist/mini-html-webpack-plugin",
-  "repository": "styleguidist/mini-html-webpack-plugin",
-  "engines": {
-    "node": ">=8.12"
-  },
-  "devDependencies": {
-    "@webpack-contrib/test-utils": "^0.1.3",
-    "css-loader": "^3.4.2",
-    "eslint": "^5.16.0",
-    "eslint-config-tamia": "^6.0.2",
-    "husky": "^2.3.0",
-    "jest": "^24.8.0",
-    "lint-staged": "^8.1.7",
-    "mini-css-extract-plugin": "^0.9.0",
-    "prettier": "^1.17.1",
-    "webpack": "^4.31.0"
-  },
-  "lint-staged": {
-    "*.{js,md}": [
-      "prettier --write",
-      "git add"
-    ],
-    "*.js": [
-      "eslint --fix",
-      "git add"
-    ]
-  },
-  "husky": {
-    "hooks": {
-      "pre-commit": "lint-staged"
-    }
-  }
+	"name": "mini-html-webpack-plugin",
```

Comment:
Could you revert indentation changes in this file?

Reply:
I think it's coming from Prettier. I'll check.

Reply:
We shouldn't use Prettier on this file: npm will reformat it.

Reply:
True, let me see if I can configure it to exclude package.json.